src/styles/blocks/_section.scss

```diff
@@ -0,0 +1,38 @@
+.section {
+
+  &__about {
+    @include allMarginsY(92px, 132px, 164px);
+  }
+
+  &__compare {
+    @include allMarginsY(92px, 120px, 144px);
+  }
+
+  &__details {
+    @include allMarginsY(80px, 120px, 180px);
+  }
+
+  &__contacts {
+    margin: 0 0 120px;
+  }
+
+  &__title {
+    margin: 0 0 48px;
+    text-align: center;
+
+    @include allFontSize(32px, 48px, 64px);
+    font-weight: 500;
+    line-height: 100%;
+    letter-spacing: -1.5px;
+  }
+
+  &__sub-title {
```

Comment:
Line height is missing, making the lines stick on top of each other on mobile especially

Reply:
Wait you're right. Must've reset because it's h3